include/ajax.tasks.php

```diff
@@ -336,20 +336,18 @@ function massProcess($action, $w=null)  {
                 $info[':title'] = sprintf('Reopen %s',
                          _N('selected task', 'selected tasks', $count));
```

Comment:
@greezybacon 
All over ajax.tasks.php and ajax.tickets.php you're using the combination of words like "Claim", "Assign", "Reopen",... and "x selected task(s)/ticket(s)" which isn't a big problem in English, but in German it just looks a bit weird since the words often would need another form, so e.g. "Zuweisen 1 ausgewähltes Ticket" would sound and look better if it were "1 ausgewähltes Ticket **Z**uweisen", but as you note the issue then is uppercase/lowercase.

My suggestion to solve this is to make it as it is when changing the status of a ticket.
There it's "Change Status &mdash; 1 selected Ticket". Because of "&mdash;" it's like these are 2 separated short strings who look ok and not 1 longer string who looks weird (at least in German).

What do you mean? Would that be a solution to add "&mdash;" (`&mdash;`) to all those :title's?